pkg/generate/testdata/models/apis/elasticache/0000-00-00/generator.yaml

```diff
@@ -1,9 +1,25 @@
 resources:
   Snapshot:
     spec_fields:
-        -
-          operation_id: CopySnapshot
-          member_name: SourceSnapshotName
+      - operation_id: CopySnapshot
+        member_name: SourceSnapshotName
+  CacheParameterGroup:
+    exceptions:
+      terminal_codes:
+        - CacheParameterGroupAlreadyExists
+        - CacheParameterGroupQuotaExceeded
+        - InvalidCacheParameterGroupState
+        - InvalidGlobalReplicationGroupState
+        - InvalidParameterCombination
+        - InvalidParameterValue
+    spec_fields:
+      - operation_id: ModifyCacheParameterGroup
+        member_name: ParameterNameValues
+    status_fields:
+      - operation_id: ModifyCacheParameterGroup
+        member_name: ParameterNameValues
```

Comment:
We should not have the same-named field in both the Spec and the Status...

Reply:
removed the field from Status, however kept the changes in generator logic for status field as it is needed for upcoming changes.